### PR TITLE
Added build commands, updated README accordingly

### PR DESCRIPTION
**Pull request by bensleveritt**
Should make it a little easier for new installers.

As per #283

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,10 @@
     "prod": "electron electron --debug=5858 --devtools --enable-logging --env=https://app.wire.com",
     "localhost": "electron electron --debug=5858 --devtools --enable-logging --env=http://localhost:8888",
     "test": "eslint electron && electron-mocha tests",
-    "rebuild-native-modules": "electron-rebuild -m ./electron/node_modules -f"
-
+    "rebuild-native-modules": "electron-rebuild -m ./electron/node_modules -f",
+    "build:macos": "grunt macos-prod",
+    "build:win": "grunt win",
+    "build:linux": "grunt linux"
```
Comment (lipis) [inline package.json]: `grunt win-prod` and `grunt linux-prod` as it's in the original.

**Comment by lipis** [comment]
I added a little comment.. but yes! Find by me.. let's do this.

**Comment by bensleveritt** [comment]
Well spotted, updated accordingly.

**Comment by lipis** [comment]
Thank you @bensleveritt! Sorry for the delay :)

**Pull request merged**